functions.md

```diff
@@ -531,6 +531,30 @@ flatten([1, [2, [3, [4, []]]]]) => [1,2,3,4]
 flatten(null)                   => null
 ```
 
+### _all(array) -> boolean_
+
+True iff all elements of `array` evaluates to `true`
+
+Examples:
+
+```
+all([true, true, true])         => true
+all([true, true, false])        => false
+all(null)                       => false
+```
+
+### _any(array) -> boolean_
+
+True iff any elements of `array` evaluates to `true`.
+
+Examples:
+
+```
+any([false, false, false])      => false
+any([false, false, true])       => false
+any(null)                       => false
```

Comment:
`null` should give `null` here, too.

We also need to decide on `any([])`, but in this case I think it's easy. If there are no `true` values then clearly the results should be `false`